components/table-of-contents/src/Section/index.js

```diff
@@ -11,32 +11,39 @@ export class Entry extends Component {
     toc: PropTypes.shape({
       addContent: PropTypes.func.isRequired,
       removeContent: PropTypes.func.isRequired,
-    }).isRequired,
+    }),
   }
 
   componentDidMount() {
-    this.props.toc.addContent(this.props.id, this.props.title);
+    if(this.props.toc) {
```

Comment:
is this not a linting error? i.e. there should be a space in here like this: `if (this.props.toc) {`

Reply:
I can change that - this branch still has all the linting overrides to enable previous code to pass, and has the glob that doesn't work.